symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -83,11 +83,31 @@ void test_basic() {
     basic_free_heap(y);
 }
 
+void test_complex() {
+    basic e;
+    basic f;
+    char* s;
+    basic_init(e);
+    basic_init(f);
```

Comment:
That's weird. Can you start from scratch, i.e. save your work, then do `git clean -dfx`, and then post the full output of
```
cmake -DCMAKE_BUILD_TYPE=Debug
make VERBOSE=1
```
This will print a lot of stuff, and if you post it, we should be able to figure out what's causing it. Also post platform and compiler version. Just create a new issue for it.

The idea is that if cmake succeed, then it should never fail like this during `make`. So we probably have to fix a bug in cmake somewhere.